docs/develop/scripting-fundamentals.md

```diff
@@ -64,7 +64,7 @@ A range is a group of contiguous cells in the workbook. Scripts typically use A1
 
 Ranges have three core properties: values, formulas, and format. These properties get or set the cell values, formulas to be evaluated, and the visual formatting of the cells. They are accessed through `getValues`, `getFormulas`, and `getFormat`. Values and formulas can be changed with `setValues` and `setFormulas`, while the format is a `RangeFormat` object comprised of several smaller objects that are individually set.
 
-Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section of Using built-in JavaScript objects in Office Scripts](javascript-objects.md#working-with-ranges) for more information on handling those arrays in the Office Scripts framework.
+Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section](javascript-objects.md) of [Built-in JavaScript objects in Office Scripts](javascript-objects.md#working-with-ranges) for more information on handling those arrays in the Office Scripts framework.
```

Comment:
@lindalu-MSFT, what's the preferred way to link to an article section?

Reply:
To link to a bookmark in a different article, I'd do this:

For more information on handling arrays in the Office Scripts framework, see [Working with ranges](javascript-objects.md#working-with-ranges). 

It's not necessary to include the link to the source topic, unless you think it will add further clarity. Keep it simple.